src/wui/game_main_menu_save_game.cc

```diff
@@ -45,11 +45,15 @@ GameMainMenuSaveGame::GameMainMenuSaveGame(InteractiveGameBase& parent,
                                            const Type type)
    : UI::UniqueWindow(&parent,
                       UI::WindowStyle::kWui,
-                      type == Type::kSave ? "save_game" : "load_game",
+                      type == Type::kSave       ? "save_game" :
+                      type == Type::kLoadReplay ? "load_replay" :
+                                                  "load_game",
                       &registry,
                       parent.get_inner_w() - 40,
                       parent.get_inner_h() - 40,
-                      type == Type::kSave ? _("Save Game") : _("Load Game")),
+                      type == Type::kSave       ? _("Save Game") :
+                      type == Type::kLoadReplay ? _("Load Replay") :
+                                                  _("Load Game")),
```

Comment:
These nested ternary operators look ugly and are hard to read. Maybe define `map`s for conversion?
(with `gettext_noop()` for `_()`)

Reply:
done